code/style.css

```diff
@@ -1,7 +1,214 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #fff;
+}
+
+.header-section {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+video {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  z-index: -1;
+}
+
+.hero-image-section {
+  margin: 0;
+}
+
+/* Paragraph and text  */
+
+h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  width: 280px;
+  text-align: center;
+  margin-bottom: 150px;
+}
+
+h3 {
+  font-size: 2rem;
```

Comment:
Nice that you are working with rem, I have not learned it fully yet so will try it next time! Thanks!